tests/terraform/graph/checks/resources/APIProtectedByWAF/main.tf

```
@@ -62,7 +62,18 @@ resource "aws_api_gateway_stage" "pass" {
   stage_name    = "example"
 }
 
+resource "aws_api_gateway_stage_wafv2" "pass" {
```

```suggestion
resource "aws_api_gateway_stage" "wafv2_pass" {
```
that's what you probably meant, right?

---

and don't forget to change it in `expected.yml` file 😄

---

Hahaha, you are totally right. Sorry! Fixing it now.